store/cachekv/internal/mergeiterator.go

```diff
@@ -14,18 +14,18 @@ import (
 // cache shadows (overrides) the parent.
 //
 // TODO: Optimize by memoizing.
-type cacheMergeIterator struct {
+type CacheMergeIterator struct {
```

Comment:
given that all fields are private, I don't understand why making this public. can we ignore?

Reply:
Sure